app/commands/user/set_discord_roles.rb

```diff
@@ -23,6 +24,12 @@ def set_supermentor_role!
     set_role!(SUPERMENTOR_ROLE_ID)
   end
 
+  def set_insiders_role!
+    return unless user.insider?
+
+    set_role!(INSIDERS_ROLE_ID)
+  end
```

Comment:
We need the inverse of this too. If they're not an insider any more, we need to remove the role.